README.md

```diff
@@ -76,11 +77,13 @@ val = rand(500,5, 2)
 chn = Chains(val, ["a", "b", "c", "d", "e"])
 ```
 
-By default, parameters will be given the name `Parami`, where `i` is the parameter number.
+By default, parameters will be given the name `:parami`, where `i` is the parameter number.
```

Comment:
I don't have a strong opinion on this. I had the impression a common practice in Julia is to avoid underscores if it doesn't impact readability (e.g., `isconcretetype` etc.) but I'm not sure if it's more readable with or without underscore here.

Reply:
I think it's slightly more readable since it's a number on the right hand side, but that's just me.

Reply:
As I said, I don't have a strong opinion on this one, so I'll change it.